docs/js/alert.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The Alert component is an empty container where you can add a message that will be presented to users. It has an OK button by default.</p>
+
+An alert box is often used when you want to make sure information comes through to the user.
+
+### How to use
+
+You can initiate the component as a jQuery plugin:
+
+```js
+// using any selector
+$('any-selector').on('click', () => $.fn.alert(options));
+```
+
+or a vanilla constructor:
+
+```js
+import Alert from 'garden/src/js/components/alert';
+
+new Alert(options);
+```
+
+## Options
+
+Alert provides some customizable options such as: `textMessage`, `textButton`, `size`, and `triggerClose`. The default value for the `textMessage` is 'This is an example message', for the `textButton` is 'Ok', the `size` value is 'medium', and `triggerClose` is 'data-alert-button'.
+
+| Option            | Default | Description |
+|-------------------|-------------|
+| textMessage  | `This is an example message` | A text to display in the alert |
+| textButton | `Ok` | A text to alert button |
+| size | `"medium"` | The modal size may vary between small, large, or medium |
+| triggerClose | `data-alert-button` | A string selector to bind and call hide method when clicked |
+
+
+Ex:
+
+```js
+let options = {
+  textMessage: 'This is an example message',
+  textButton: 'Ok'
+}
+
+// as a jquery plugin
+$('[data-alert]').on('click', () => $.fn.alert(options));
+
+// as a vanilla constructor
+new Alert(options);
+```
+
+## Working with alert
+
+By default, alert provides a text message, a text to the button and, his size. Here's an example of how to create a button to open a alert.
```

Comment:
Que tal ajustar pra `an alert` tanto no começo quanto no final da oração?
A vírgula é antes do `and`
Troca `of` por `on` na segunda oração.

No inglês existe algo chamado [Parallel Structure](https://owl.english.purdue.edu/owl/resource/623/01/), então quando tu começa enumerar itens seguindo uma estrutura tu precisa terminar com a mesma estrutura.
Nesse caso tu começou com `a text message`, `a text to the button` e no último já usou outra estrutura. Talvez mudando o último item para `..., and a size value`. O que acha?